sdk/eventgrid/eventgrid-namespaces/samples-dev/namespaceActivities.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Publish and Receive events to Event Grid.
+ * @azsdk-weight 3
+ */
+
+import { EventGridClient, CloudEvent, ReceiveResult } from "@azure/eventgrid-namespaces";
+import { AzureKeyCredential } from "@azure/core-auth";
+
+import * as dotenv from "dotenv";
+
+// Load the .env file if it exists
+dotenv.config();
+
+const endpoint = process.env["EVENT_GRID_NAMESPACES_ENDPOINT"] ?? "https://endpoint";
+const key = process.env["EVENT_GRID_NAMESPACES_KEY"] ?? "api_key";
+const eventSubscripionName = process.env["EVENT_SUBSCRIPTION_NAME"] ?? "testsubscription1";
+const topicName = process.env["TOPIC_NAME"] ?? "testtopic1";
```

Comment:
Add `EVENT_SUBSCRIPTION_NAME` and `TOPIC_NAME` to sample.env?

Reply:
Done(in next commit)